v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/CassandraAllDataTypesIT.java

```diff
@@ -55,14 +74,79 @@ public class CassandraAllDataTypesIT extends SourceDbToSpannerITBase {
   private static final Logger LOG = LoggerFactory.getLogger(MySQLDataTypesIT.class);
   private static PipelineLauncher.LaunchInfo jobInfo;
 
-  public static CassandraResourceManager cassandraResourceManager;
-  public static SpannerResourceManager spannerResourceManager;
+  private static CassandraResourceManager cassandraResourceManager;
+  private static SpannerResourceManager spannerResourceManager;
+  private static final List<FlexTemplateDataflowJobResourceManager>
+      dlqFlexTemplateDataflowJobResourceManagers = Collections.synchronizedList(new ArrayList<>());
 
   private static final String CASSANDRA_DUMP_FILE_RESOURCE =
       "DataTypesIT/cassandra-data-types.csql";
 
   private static final String SPANNER_DDL_RESOURCE = "DataTypesIT/cassandra-spanner-schema.sql";
 
+  private FlexTemplateDataflowJobResourceManager launchDlqReplay(LaunchInfo bulkJobInfo)
+      throws IOException, InterruptedException {
+    String dlqGcsPath = getDlqPath(bulkJobInfo);
+    String dlqJobName = PipelineUtils.createJobName("dlq-" + getClass().getSimpleName());
+    LOG.info(
+        "Bulk Job ID = {}, DLQ Job Name = {}, bulkInfo = {}",
+        bulkJobInfo.jobId(),
+        dlqJobName,
+        bulkJobInfo);
+
+    createAndUploadJarToGcs("CustomTransformationAllDataTypes");
```

Comment:
nit: store `CustomTransformationAllDataTypes` in a constant , got confused a bit

Reply:
Fixed in current patch.